### PR TITLE
Add Tcl/Tk 8.6 to manylinux2014

### PR DESCRIPTION
**Pull request by marcelotduarte**
I'm using manylinux2014 to generate wheels for cx_Freeze. As Tcl/Tk 8.5 is still used, it ended up creating an issue (https://github.com/marcelotduarte/cx_Freeze/issues/1577). So I upgrade to Tcl/Tk 8.6 in the same way as sqlite3.

### CODE REVIEW

**Comment by marcelotduarte** [comment]
@auvipy The [build](https://github.com/pypa/manylinux/actions/runs/2826116863) passed.
But Travis returned "The job exceeded the maximum time limit for jobs, and has been terminated."

**Review by auvipy — APPROVED** [review]

**Comment by marcelotduarte** [comment]
ping @mayeut

**Comment by mayeut** [comment]
> Travis returned "The job exceeded the maximum time limit for jobs, and has been terminated."

This was just a glitch. I triggered a build on travis-ci without any cache and build times are OK.

File: docker/Dockerfile [hunk 1]
```diff
@@ -95,6 +95,13 @@ RUN export OPENSSL_ROOT=openssl-1.1.1q && \
     export OPENSSL_DOWNLOAD_URL=https://www.openssl.org/source && \
     manylinux-entrypoint /build_scripts/build-openssl.sh
 
+COPY build_scripts/build-tcltk.sh /build_scripts/
+RUN export TCL_ROOT=tcl8.6.12 && export TK_ROOT=tk8.6.12 \
+    export TCL_HASH=26c995dd0f167e48b11961d891ee555f680c175f7173ff8cb829f4ebcde4c1a6 && \
+    export TK_HASH=12395c1f3fcb6bed2938689f797ea3cdf41ed5cb6c4766eec8ac949560310630 && \
```
Comment (mayeut) [inline docker/Dockerfile:101]: This needs to be updated on patch releases of TCL/TK.
Please update `update_native_dependencies.py` to handle this.

**Review by mayeut — CHANGES_REQUESTED** [review]
tcl/tk default runtime is still being installed in:
https://github.com/pypa/manylinux/blob/92d31314bc6880b33ebc76cf2d95ac36b7114d84/docker/build_scripts/install-runtime-packages.sh#L48
and devel packages in:
https://github.com/pypa/manylinux/blob/92d31314bc6880b33ebc76cf2d95ac36b7114d84/docker/build_scripts/install-build-packages.sh#L17

Those should be updated not to bloat image size with something that's unused.

File: docker/Dockerfile (hunk 1 above)
Comment (marcelotduarte) [reply docker/Dockerfile]: @mayeut Apparently I cannot update from sourceforge using lastversion. I tied via command line too.
The initial work for this is at https://github.com/marcelotduarte/manylinux/commit/823e114ca421448f877d1cdb8078509f29ca58d1
Can you take a look?

In the SF project site it shows 8.6.11 as the last version:
image

Comment (mayeut) [reply docker/Dockerfile]: You can use tcl/tk GitHub mirror to check the latest version.
```
lastversion tcltk/tcl --only core-8-6-
8.6.12
```

Comment (marcelotduarte) [reply docker/Dockerfile]: Wow. Thanks!
Done.

**Comment by mayeut** [comment]
Thanks @marcelotduarte !

**Pull request merged**